data/hai/hai missions.txt

```diff
@@ -2280,8 +2236,7 @@ mission "Strider notification"
 	to offer
 		has "event: pond strider mass production"
 	on offer
-		log `For assisting the Hai establish an alien embassy, they've made newly developed warships and outfits available.`
 		conversation
 			scene `thumbnail/hai pond strider`
-			`The Hai thank you for your assistance with interspecies diplomacy. They have begun production of some new military ships and outfits that they've granted you access to in their shipyards and outfitters.`
+			`Upon landing on <planet>, you receive a message from Torsa. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design, and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoiler the surprise for you, though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`
```

Comment:
```suggestion
			`Upon landing on <planet>, you receive a message from Torsa. "Hello Captain <last>! I hope this message finds you well. I would like to once again thank you for your help on the Pond Strider, and would like to inform you that we have finished implementing the hull repair technology that you have provided us into the Pond Strider design and have now begun producing the ship on a mass scale. You'll also be interested to find that we have developed new weaponry for the Pond Strider's drone, the Flea. I won't spoiler the surprise for you though. Check the shipyard on Hai-home the next time you stop by to take a look at it."`
```
Both the commas in the extra-long sentence I removed one from are optional. I removed the one I did because I think it helps a reader interpret the structure and phrasing a bit better.